test/system/160-volumes.bats

```diff
@@ -64,6 +64,29 @@ function teardown() {
 }
 
 
+# Filter volumes by name
+@test "podman volume filter --name" {
+    suffix=$(random_string)
+    prefix="volume"
+
+    for i in 1 2; do
+        myvolume=${prefix}_${i}_${suffix}
+        run_podman volume create $myvolume
+        is "$output" "$myvolume" "output from volume create $i"
+    done
+
+    run_podman volume ls --filter name=${prefix}_1.+ --format "{{.Name}}"
+    is "$output" "${prefix}_1_${suffix}" "--filter name=${prefix}_1.+ shows only one volume"
+
+    # The _1* is intentional as asterisk has different meaning in glob and regexp. Make sure this is regexp
+    run_podman volume ls --filter name=${prefix}_1* --format "{{.Name}}"
+    is "$output" "${prefix}_1_${suffix}" "--filter name=${prefix}_1* shows only one volume"
```

Comment:
This is EXACTLY why I requested the change I requested: because regexps are counterintuitive to many people. `_1*` will (and should) match **BOTH** volumes, because `1*` means "zero or more instances of the character 1`.

Reply:
yes we are, docker uses regex and we also use regex for all other commands.
I have no idea where the glob idea comes from, anyway this is the filter line in moby: https://github.com/moby/moby/blob/c17a566b0dc5074de8ce25a88ba1d43e10667d19/api/types/filters/parse.go#L190

Reply:
Yep, my bad. I accidentally copied the wrong line. I updated the expected results.